ddtrace/contrib/internal/openai/utils.py

```diff
@@ -260,38 +269,47 @@ def _loop_handler(span, chunk, streamed_chunks):
     When handling a streamed chat/completion/responses,
     this function is called for each chunk in the streamed response.
     """
-
     if span.get_tag("openai.response.model") is None:
         if hasattr(chunk, "type") and chunk.type.startswith("response."):
             response = getattr(chunk, "response", None)
             model = getattr(response, "model", "")
         else:
             model = getattr(chunk, "model", "")
         span.set_tag_str("openai.response.model", model)
-    # Only run if the chunk is a completion/chat completion
+
+    for response in getattr(chunk, "response", ""):
```

Comment:
Why are we iterating `for response in chunk.response` here? I thought there is only one response obejct per chunk.

Reply:
I think we can just rely on the `type='response.completed'` chunk, which contains all output data we care about putting on the response span. However it seems like a few different chunks also contain a `response` field which are empty, so we'll need to check for the chunk type here.